images/router/haproxy/conf/haproxy-config.template

```diff
@@ -231,6 +231,29 @@ backend be_edge_http_{{$cfgIdx}}
   timeout server  {{$value}}
       {{ end }}
     {{ end }}
+
+{{ if index $cfg.Annotations "router.openshift.io/haproxy.rate-limit-connections" }}
```

Comment:
I think <router-name>.router.openshift.io/<feature> sounds better scoped. @JacobTanenbaum change the one that I wrote too then maybe (will need testcase changes too).